app/models/user.rb

```diff
@@ -1,8 +1,16 @@
 class User < ActiveRecord::Base
 
+  attr_accessor :password_confirmation
   has_secure_password validations: false
 
   validates_presence_of :email, :password, :full_name
   validates_uniqueness_of :email
+  validate :password_equality
+
+  def password_equality
```

Comment:
If you want to have validation on confirmation, you may as well remove `validations: false` from `has_secure_password`. Or validate on confirmation - Study docs [here](http://guides.rubyonrails.org/active_record_validations.html#confirmation).